milestone_1/report _milestone_1.md

```diff
@@ -17,6 +17,25 @@ git clone https://github.com/FLX-20/Data-Science-Toolkits-and-Architectures-grou
 ```
 It clones/copes all files, commits history and branches from the repository located in the cloud on Git Hub into a new directory on your computer. Thus it creates a local copy of the project.
 
+### 3.2 Creating Branches
+If you want to make changes in the repo it is a good habit to create a new branch. It is not recommended that changes be made directly at the main branch. You can create a new branch with the following command.
+```
+git branch <name_of_branch>
+```
+Check if the branch was successfully create with the next command, which returns all branches in your current local repo.
+```
+git branch
+```
+You can change the branch, in which your are currently working with:
+```
+git switch <name_of_branch>
+```
+If you want to delete a branch use the next command.
+```
+git branch -d <branch_name>
+```
+
```

Comment:
Do : after each sentence before you give a name of a code or . to be consistent because you do both.